src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -272,6 +282,15 @@ export class WorkspaceSavedObjectsClientWrapper {
       objects: Array<SavedObjectsBulkCreateObject<T>>,
       options: SavedObjectsCreateOptions = {}
     ): Promise<SavedObjectsBulkResponse<T>> => {
+      // Only OSD admin can bulkCreate workspace
+      if (
+        isTypeContained(
+          WORKSPACE_TYPE,
+          objects.map((object) => object.type)
+        )
+      )
```

Comment:
Nit: will this look a bit cleaner?
```suggestion
      // Only OSD admin can bulkCreate workspace
      if (objects.some(obj => obj.type === WORKSPACE_TYPE))
```

Reply:
Sure, I will simply the code.